service/controller/aws/v8/key/key.go

```diff
@@ -13,12 +13,13 @@ func ChartSpecs() []key.ChartSpec {
 	// Add any provider specific charts here.
 	return []key.ChartSpec{
 		{
-			AppName:       "coredns",
-			ChannelName:   "0-1-stable",
-			ChartName:     "kubernetes-coredns-chart",
-			ConfigMapName: "coredns-values",
-			Namespace:     metav1.NamespaceSystem,
-			ReleaseName:   "coredns",
+			AppName:           "coredns",
+			ChannelName:       "c5bea1db131f72cfee47038fefcd65dece13cc62-basic",
```

Comment:
Note: revert this to `0-2-stable` before merging.

Reply:
FYI (or next time) you can reference `wip-SHA` here. See: https://github.com/giantswarm/kubernetes-coredns/pull/13#issuecomment-431340178